src/api/spec/features/webui/interconnects_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'browser_helper'
+
+RSpec.feature 'Interconnects', type: :feature, js: true, vcr: true do
```

Comment:
What do we need vcr for? And where are the cassettes?

Reply:
to write the project in the backend, don't we? (cassettes coming :rocket:)

Reply:
If you would not visit it again you don't need it to be written to the backend...

Reply:
but when you create a custom one it is redirecting there

Reply:
okay true...

Reply:
maybe it shouldn't... but that is the current behavior...